### PR TITLE
:recycle: The displayed image switches according to Dark/Light mode.

### LINKED ISSUE #1032: [Idea] Switch the top image of the README according to the LIGHT/DARK mode.
**Idea Description**
- I thought it would be interesting to switch the top image using [this feature](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to) of Markdown.
- A dark mode version of the top image in the README can be reflected immediately when available.

**Reference images and links**

The image below switches display depending on the terminal's LIGHT/DARK mode.

<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/DroidKaigi/conference-app-2023/main/feature/about/src/main/res/drawable-night-xxhdpi/img_about_key_visual.webp">
  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/DroidKaigi/conference-app-2023/main/feature/about/src/main/res/drawable-xxhdpi/img_about_key_visual.webp">
  Shows an illustrated sun in light mode and a moon with stars in dark mode.
</picture>

### PR DESCRIPTION
**Pull request by Corvus400**
## Issue
- close #1032 

## Overview (Required)
- The title is telling.

## Links
- https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to

## Screenshot (Optional if screenshot test is present or unrelated to UI)
Before | After
:--: | :--:
 |

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
-![readme-banner](https://github.com/DroidKaigi/conference-app-2023-images/assets/1386930/22efb1a7-5a43-469c-a493-fa619034863e)
-
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/136104152/264391682-ae2b1067-966f-47a2-aa10-4e1ee4d09418.png">
```
Comment (Corvus400) [inline README.md:2]: The image of Dirk is
https://github.com/DroidKaigi/conference-app-2023-images/assets/
It is believed that they need to be placed under the distribution.

Comment (Corvus400) [reply README.md:2]: @takahirom 
Can you please put the Dark image in the same place as the Light and give us the URL?🙇‍♂️

Comment (takahirom) [reply README.md:2]: The reason we stored the image in a separate repository is as follows:

* We were developing in a private repository.
* Images added while the repository is private remain invisible even after making the repository open source.


You can now add images to the conference-app-2023 repository. Simply drag and drop the image into GitHub's web editor to do so.

Comment (Corvus400) [reply README.md:2]: @takahirom 
I see!
Then it looks like we should keep the current references, so we will remove the Draft status of PR.

**Review by takahirom — APPROVED** [review]
Thanks for adaptive experience for developers!

**Pull request merged**